app/controllers/application_controller.rb

```diff
@@ -15,4 +23,8 @@ def set_locale
   def default_url_options(options = {})
     { locale: I18n.locale }.merge options
   end
+
+  def added_attrs
+    %I[name email password password_confirmation remember_me]
```

Comment:
fyi: 
`%i` - haven't interpolation - same as `'`
`%I` - have interpolation - same as `"`
This rule also works for `%w`, `%W`, `%q`, `%Q`